api.md

```diff
@@ -132,6 +132,8 @@ You can also attach to named parent properties using `attachObject={[target, nam
   <bufferAttribute attachObject={['attributes', 'position']} count={v.length / 3} array={v} itemSize={3} />
 ```
 
+**New in v5**, all elements ending with "Material" receive `attach="material"`, and all elements ending with "Geometry" receive `attach="geometry"` automatically. Of course you can still overwrite it, but it isn't necessary to type out any longer.
```

Comment:
I would perhaps move this up to be above or below line 110.

Reply:
do you have access to the v5 branch? you can just do stuff like that no questions asked - im pretty bad with keeping code and comments clean 😞

Reply:
I do, I will do it :-)